docs/source/development-guide/schema-deployment/deploy-test-process.rst

```diff
@@ -0,0 +1,15 @@
+Deployment Test Process
```

Comment:
This document does not belong under `schema-deployment`, as it's not related to the schema specifically.

Reply:
Additionally, some of this information is already covered in the repository's `CONTRIBUTING.md`, so we need to pick either that file or this file for defining contribution workflows, instead of maintaining both.